Agreements/External/PN_D.txt

```diff
@@ -0,0 +1,30 @@
+FIR_COPX:*:*:SOSUN:EGGW:*:LFFF Low (245-295):London AC Dover:*:28000:SOSUN
+FIR_COPX:*:*:SOSUN:EGHI:*:LFFF Low (245-295):London AC Dover:*:28000:SOSUN
+FIR_COPX:*:*:SOSUN:EGHH:*:LFFF Low (245-295):London AC Dover:*:28000:SOSUN
+FIR_COPX:*:*:SOSUN:EGLF:*:LFFF Low (245-295):London AC Dover:*:28000:SOSUN
+FIR_COPX:*:*:SOSUN:EGSC:*:LFFF Low (245-295):London AC Dover:*:28000:SOSUN
+FIR_COPX:*:*:SOSUN:EGSS:*:LFFF Low (245-295):London AC Dover:*:28000:SOSUN
+
+FIR_COPX:*:*:KESAX:EGKK:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+
+FIR_COPX:*:*:KESAX:EGHL:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+FIR_COPX:*:*:KESAX:EGLF:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+FIR_COPX:*:*:KESAX:EGLK:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+FIR_COPX:*:*:KESAX:EGHI:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+FIR_COPX:*:*:KESAX:EGHH:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+FIR_COPX:*:*:KESAX:EGTF:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+FIR_COPX:*:*:KESAX:EGVO:*:LFFF Low (245-295):London AC Dover:*:28000:KESAX
+
+FIR_COPX:*:*:KESAX:EGKA:*:LFFF Low (DB-245):London AC Dover:*:20000:KESAX
+
+FIR_COPX:*:*:KESAX:EGGW:*:LFFF Low (245-295):London AC Dover:*:26000:KESAX
+FIR_COPX:*:*:KESAX:EGSS:*:LFFF Low (245-295):London AC Dover:*:26000:KESAX
+FIR_COPX:*:*:KESAX:EGSC:*:LFFF Low (245-295):London AC Dover:*:26000:KESAX
+
+FIR_COPX:*:*:ABNUR:EGLL:*:LFFF Low (245-295):London AC Dover:*:28000:ABNUR
+FIR_COPX:*:*:ABNUR:EGWU:*:LFFF Low (245-295):London AC Dover:*:28000:ABNUR
+
+FIR_COPX:*:*:RINTI:*:*:LFFF Low (DB-245):London AC Dover:*:*:RINTI
+FIR_COPX:*:*:L15FIR:*:*:LFFF Low (DB-245):London AC Dover:*:*:L15FIR
```

Comment:
Cannot find 'L15FIR' anywhere else in the repo - any clues?

Reply:
Yeah this was an old one - I'll change it now! Also made a few further changes to RINTI/MOTOX COPX